group-generators/generators/degenscore-over-900/index.ts

```diff
@@ -0,0 +1,29 @@
+import { dataProviders } from "@group-generators/helpers/data-providers";
+import { GroupWithData, Tags, ValueType } from "topics/group";
+import {
+  GenerationContext,
+  GenerationFrequency,
+  GroupGenerator,
+} from "topics/group-generator";
+
+const generator: GroupGenerator = {
+  generationFrequency: GenerationFrequency.Weekly,
+
+  generate: async (context: GenerationContext): Promise<GroupWithData[]> => {
+    const degenscoreProvider = new dataProviders.DegenScoreProvider();
+    const addresses = await degenscoreProvider.getBeaconOwnersWithScore(900);
```

Comment:
You need to change this part of your group, `getBeaconOwnersWithScore` require a type `Score`

Reply:
fixed with https://github.com/sismo-core/sismo-hub/pull/1555/commits/b3ba22fdf668b90d64dd038267c881c1596e3869